regulations/views/notice_home.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from operator import itemgetter
+
+import logging
+
+from django.http import Http404
+from django.template.response import TemplateResponse
+from django.views.generic.base import View
+
+from regulations.generator.api_reader import ApiReader
+from regulations.views.preamble import (
+    notice_data, CommentState)
+
+
+logger = logging.getLogger(__name__)
+
+
+class NoticeHomeView(View):
+    """
+    Basic view that provides a list of regulations and notices to the context.
+    """
+    template_name = None  # We should probably have a default notice template.
+
+    def get(self, request, *args, **kwargs):
+        notices = ApiReader().notices().get("results", [])
+        context = {}
+        notices_meta = []
+
+        for notice in notices:
+            try:
+                if notice.get("document_number"):
+                    _, meta, _ = notice_data(notice["document_number"])
+                    notices_meta.append(meta)
+            except Http404:
+                pass
+
+        notices_meta = sorted(notices_meta, key=itemgetter("publication_date"),
+                              reverse=True)
+
+        context["notices"] = notices_meta
+        # Django templates won't show contents of CommentState as an Enum, so:
```

Comment:
Yeah, I [was](https://github.com/eregs/regulations-site/pull/382/commits/36c2fc9b4ba110c94a4be52ab9556a2bb3661afa) torn on this. The problem is that Django templates will silently fail if there's a typo in the `CommentState` value, so they're not getting us anything over a string comparison. The one benefit to a string comparison is not needing to add it to the template's context. That could also be accomplished by including it as a global template var, though?

I don't know the right answer, but we should be consistent.